subprojects/docs/src/docs/release/notes.md

```diff
@@ -58,6 +58,17 @@ This improves the likelihood of [build cache hits](userguide/build_cache.html) w
 See the [userguide](userguide/more_about_tasks.html#sec:meta_inf_normalization) for further information.  Note that this API is incubating and will likely change in future releases as support 
 is expanded for normalizing properties files outside of `META-INF`.
 
+## Improvements for plugin authors
+
+### New ZIP and TAR `FileTree` factory methods
+
+Previously, it was only possible to create a `FileTree` for a ZIP or TAR archive by using the APIs provided by a `Project`.
+However, a `Project` object is not always available.
```

Comment:
If a `Project` object isn't available, then a `ProjectLayout` is also unlikely to be available.